File: persephone/provisioning-service/src/main/java/com/vmware/blockchain/deployment/services/orchestration/model/vsphere/OvfProperty.java
/*
 * Copyright (c) 2020 VMware, Inc. All rights reserved. VMware Confidential
 */

package com.vmware.blockchain.deployment.services.orchestration.model.vsphere;

import lombok.Builder;
import lombok.Data;

/**
 * Simplified binding of OVF property parameter.
 */
@Data
@Builder
public class OvfProperty {
    String id;
    String value;
}

File: persephone/orchestration-vmware/src/common/com/vmware/blockchain/deployment/model/vsphere/OvfProperty.kt
/* **************************************************************************
 * Copyright (c) 2019 VMware, Inc.  All rights reserved. VMware Confidential
 * *************************************************************************/
package com.vmware.blockchain.deployment.model.vsphere

import kotlinx.serialization.ContextualSerialization
import kotlinx.serialization.Serializable

/**
 * Simplified binding of OVF property parameter.
 */
@Serializable
data class OvfProperty(val id: String,
                       @ContextualSerialization
                       val value: Any)
